src/icon.js

```diff
@@ -1,8 +1,17 @@
 const path = require('path');
 
-module.exports = {
-  done: path.join(__dirname, 'icons/macos/doneTemplate.png'),
-  todo: path.join(__dirname, 'icons/macos/todoTemplate.png'),
-  load: path.join(__dirname, 'icons/macos/loadTemplate.png'),
-  fail: path.join(__dirname, 'icons/macos/failTemplate.png')
-};
+if (process.platform == 'win32') {
+  module.exports = {
+    done: path.join(__dirname, 'icons/win/doneTemplate.png'),
```

Comment:
Icons only need to be suffixed with `Template` on macOS, this is for light and dark mode support - for Windows we can remove them.

I noticed that the tray icons look quite pixelated in your screenshot, according to the [Electron docs](https://electronjs.org/docs/api/tray#class-tray):
> On Windows it is recommended to use ICO icons to get best visual effects.